pkg/auth/mutual_authhandler.go

```diff
@@ -53,12 +53,15 @@ func newMutualAuthHandler(logger logrus.FieldLogger, lc hive.Lifecycle, cfg Mutu
 }
 
 type MutualAuthConfig struct {
-	MutualAuthListenerPort int `mapstructure:"mesh-auth-mutual-listener-port"`
+	MutualAuthListenerPort   int           `mapstructure:"mesh-auth-mutual-listener-port"`
+	mutualAuthConnectTimeout time.Duration `mapstructure:"mesh-auth-mutual-Connect-timeout"`
```

Comment:
And we should capitalize the field itself `mutualAuthConnectTimeout` -> `MutualAuthConnectTimeout`